.github/actions/conventional-checks/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Conventional Checks
```

Comment:
I guess you mean conventions and not conventional, I propose to rename to  `enforce-pr-conventions` that is more straightforward

Reply:
Done